x-pack/plugins/remote_clusters/public/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -216,28 +215,36 @@ export class RemoteClusterFormUi extends Component {
   };
 
   onSeedsInputChange = (seedInput) => {
+    if (!seedInput) {
+      // If empty seedInput ("") don't do anything. This happens
+      // right after a seed is created.
+      return;
+    }
+
     const { intl } = this.props;
-    const { fields, localSeedErrors } = this.state;
-    const { seeds } = fields;
 
-    // Allow typing to clear the errors, but not to add new ones.
-    const errors = (!seedInput || this.getLocalSeedErrors(seedInput).length === 0) ? [] : localSeedErrors;
+    this.setState(({ fields, localSeedErrors }) => {
```

Comment:
Thanks for catching these mistakes.

Reply:
No prob. First I thought that those were causing the bug of adding a remote cluster by having a race condition. But it was not 😊